SConstruct

```diff
@@ -1041,15 +1041,18 @@ env.Append(
 # using a simple prefix to maintain cross-platform compatibility.
 # For Posix, build the search path manually, for Windows stick with Scons
 # best practice and use CPPPATH
-def includeSystemDirectory( e, include_list ):
-	if type(include_list) != list:
-		include_list = [include_list]
+def formatSystemIncludes( e, includeList ) :
```

Comment:
Since this function isn't intended to modify an environment any more, we should remove the `e` argument. I see you're using it internally to get "PLATFORM", but we could do that with `sys.platform` or `Environment()["PLATFORM"]`.

Reply:
Yeah, I was waffling on that one. I'm also using it to check against empty include directories using `e.subst( i )`. I pass the specific module's environment such as `formatSystemIncludes( mayaEnv, includes)` in case in the future there are module-specific substitutions.

Do you think I should just use `env.subst( i )` instead so we can eliminate that parameter?

Reply:
> I'm also using it to check against empty include directories using e.subst( i ).

Ah, yes, I missed that. Let's leave it as you have it - thanks for the explanation.